bin/install.sh

```diff
@@ -74,13 +74,21 @@ fi
 " >> "${bashProfile}"
 fi
 
+# Suppress "Your profile name includes a 'profile ' prefix" warnings from AWS Java SDK (Resolves #233)
+loggingProperties="${HOME}/.okta/logging.properties"
+echo "com.amazonaws.auth.profile.internal.BasicProfileConfigLoader = NONE
+" > "${loggingProperties}"
+
 # Create withokta command
 echo '#!/bin/bash
 command="$1"
 profile=$2
 shift;
 shift;
-env OKTA_PROFILE=$profile java -classpath ~/.okta/okta-aws-cli.jar com.okta.tools.WithOkta $command $@
+env OKTA_PROFILE=$profile java \
+    -Djava.util.logging.config.file=~/.okta/logging.properties \
```

Comment:
@craiglink you need this change as well to make use of the logging.properties file.

Reply:
but we don't use the install.sh command to install or the withokta command. is that required now?

Reply:
That's the recommended install approach. I don't anticipate people using this without it.

Alternatively, you can edit your awscli command to add `-Djava.util.logging.config.file=~/.okta/logging.properties` to the java command arguments.